toString.js

```diff
@@ -36,10 +32,10 @@ function toString(value) {
   }
   if (Array.isArray(value)) {
     // Recursively convert values (susceptible to call stack limits).
-    return `${map(value, (other) => other == null ? other : toString(other))}`
+    return `${value.map((other) => other == null ? other : toString(other))}`
```

Comment:
i feel `()` for map parameters is redundant here, can we not use 

return `${value.map( other => other == null ? other : toString(other))}`  ?

Reply:
Its the project style. See https://github.com/lodash/lodash/pull/4418#discussion_r316724333